src/Views/HardwareView.vala

```diff
@@ -263,4 +270,26 @@ public class About.HardwareView : Gtk.Grid {
 
         return 0;
     }
+
+    private static bool laptop_detect () {
+        string? laptop_detect_executable = Environment.find_program_in_path (LAPTOP_DETECT_NAME);
+        if (laptop_detect_executable == null) {
+            warning ("laptop-detect not found");
+            return false;
+        }
+
+        int exit_status;
+
+        try {
+            Process.spawn_command_line_sync (laptop_detect_executable, null, null, out exit_status);
+            if (exit_status == 0) {
```

Comment:
You could just reduce it to `return exit_status == 0`

Reply:
Admittedly this is copy/pasted from the power plug, but I think the purpose of doing it this way is for that debug message